lib/user.rb

```diff
@@ -0,0 +1,20 @@
+require 'net/http'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.